ERP-BACKEND/tests/test_transactions.py

```
@@ -0,0 +1,504 @@
+                response = client.post(
+                    "/api/v1/crm/companies",
+                    json=child_data,
+                    headers=authenticated_headers
+                )
+                assert response.status_code == 200
+                child_id = response.json()["id"]
+                
+                # Force rollback
+                raise Exception("Simulated nested failure")
+        except Exception:
+            pass  # Expected
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)